ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -4760,4 +4760,234 @@ HWRWISOVPM:
+  is_abstract: true
+  uses:
+  - supply_fan_run_mode
+  implements:
+  - MONITORING
+
+SRC:
+  description: "Very basic run scheduling command. "
+  is_abstract: true
+  uses:
+  - schedule_run_command
+  implements:
+  - CONTROL
+
+IDSSC:
+  description: "isolation damper status monitoring on the supply side."
+  is_abstract: true
+  uses:
+  - supply_air_isolation_damper_status
+  implements:
+  - MONITORING
+
+IDSSC2X:
+  description: "isolation damper status monitoring on the supply side, where there are two separate, equal sets of dampers that operate in conjunction."
+  is_abstract: true
+  uses:
+  - supply_air_isolation_damper_status_1
+  - supply_air_isolation_damper_status_2  
+  implements:
+  - MONITORING
+
+IDSSC3X:
+  description: "isolation damper status monitoring on the supply side, where there are three separate, equal sets of dampers that operate in conjunction."
+  is_abstract: true
+  uses:
+  - supply_air_isolation_damper_status_1
+  - supply_air_isolation_damper_status_2
+  - supply_air_isolation_damper_status_3
+  implements:
+  - MONITORING   
+  
+SSPM2X:
+  description: "Static pressure monitoring on the supply side where the average of the two sensors are being taken."
+  is_abstract: true
+  uses:
+  - supply_air_static_pressure_sensor_1
+  - supply_air_static_pressure_sensor_2  
+  implements:
+  - MONITORING
+
+SFRDM:  
+  description: "Supply fan run duration monitoring "
+  is_abstract: true
+  uses:
+  - supply_fan_run_time_accumulator
+  implements:
+  - MONITORING 
+
+CPRDM:  
+  description: "Compressor run duration monitoring "
+  is_abstract: true
+  uses:
+  - compressor_run_time_accumulator
+  implements:
+  - MONITORING 
+
+CMPC:
+  description: "Compressor run control and monitoring/controlling  ."
+  is_abstract: true
+  uses:
+  - compressor_run_command
+  - compressor_run_status
+  implements:
+  - CONTROL 
+
+EVFSC:
+  description: "Evaporative fan speed monitoring/controlling  ."
+  is_abstract: true
+  uses:
+  - evaporative_fan_speed_sensor
+  implements:
+  - CONTROL 
+
+RDM:
+  description: "Very basic system run duration monitoring. "
+  is_abstract: true
+  uses:
+  - run_time_accumulator
+  implements:
+  - MONITORING
+
+HRZAM:
+  description: "High room Zone air temperature monitoring and control. "
+  is_abstract: true
+  uses:
+  - high_limit_zone_air_temperature_setpoint
+  implements:
+  - CONTROL
+  
+LRZAM:
+  description: "Low room Zone air temperature monitoring and control. "
+  is_abstract: true
+  uses:
+  - low_limit_zone_air_temperature_setpoint
+  implements:
+  - CONTROL  
+  
+HRZHM:
+  description: "High room Zone air humidity monitoring and control. "
+  is_abstract: true
+  uses:
+  - high_limit_zone_air_relative_humidity_setpoint
+  implements:
+  - CONTROL    
+  
+LRZHM:
+  description: "Low room Zone air humidity monitoring and control. "
+  is_abstract: true
+  uses:
+  - low_limit_zone_air_relative_humidity_setpoint
+  implements:
+  - CONTROL 
+
+CMPC2X:
```

Comment:
seems like this is covered by one of our many DX2 types. What does it control to?

Reply:
deleted CMPC2X and replaced by DX2ZTC